### PR TITLE
EWPP-454: Link to the RSS feed.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: oe_list_pages.module [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 declare(strict_types = 1);
 
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
```
Comment (upchuk) [inline oe_list_pages.module:10]: How come these are added here?

File: src/Plugin/ExtraField/Display/ListPageRssLink.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_list_pages\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+
+/**
+ * Extra field displaying the list page pager.
```
Comment (upchuk) [inline src/Plugin/ExtraField/Display/ListPageRssLink.php:10]: copy paste

File: src/ListBuilderInterface.php [hunk 3]
```diff
@@ -61,4 +61,15 @@ public function buildSelectedFilters(ContentEntityInterface $entity): array;
    */
   public function buildPagerInfo(ContentEntityInterface $entity): array;
 
+  /**
+   * Builds a RSS link.
```
Comment (upchuk) [inline src/ListBuilderInterface.php:65]: Builds a link to the RSS representation of the list page.

**Review by upchuk — CHANGES_REQUESTED** [review]

File: oe_list_pages.module (hunk 1 above)
Comment (sergepavle) [reply oe_list_pages.module]: Uncaught leftover.
Removed.

**Review by imanoleguskiza — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Pull request merged**